instrumentation/jaxws/jaxws-3.0-common-testing/src/main/schema/hello.xsd

```diff
@@ -16,20 +16,4 @@
       </xs:sequence>
     </xs:complexType>
   </xs:element>
-
-  <xs:element name="hello2Request">
-    <xs:complexType>
-      <xs:sequence>
-        <xs:element name="name" type="xs:string" form="unqualified"/>
-      </xs:sequence>
-    </xs:complexType>
-  </xs:element>
-
-  <xs:element name="hello2Response">
-    <xs:complexType>
-      <xs:sequence>
-        <xs:element name="message" type="xs:string" form="unqualified"/>
-      </xs:sequence>
-    </xs:complexType>
-  </xs:element>
```

Comment:
In jax-ws 2 tests `hello` request also tests the `jaxws-jws-api-1.1` instrumentation while `hello2` test is constructed so that the jws api instrumentation does not apply to it. In jax-ws 3 we don't have an instrumentation similar to the jws api instrumentation so we need only 1 test case.